autosaloon/setup_and_security.py

```diff
@@ -0,0 +1,20 @@
+KEY = 'django-insecure-nv%9k#xpi^z6dt#7rlrt5$w%epq7n6w%nwd5g*#4$nbg0a4upm'
```

Comment:
Do not store keys in code, inject them using env. Secret key leak is a high-score security vulnerability so avoid it at all cost